test/integration/rhel-8.0/multiple.test

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+SCRIPTDIR="$(readlink -f $(dirname $(type -p $0)))"
+ROOTDIR="$(readlink -f $SCRIPTDIR/../..)"
```

Comment:
As @sm00th mentioned in PR #1040, this needs to be ROOTDIR="$(readlink -f $SCRIPTDIR/../../..)"

Reply:
Thanks, will update.